cmd/lncli/walletrpc_active.go

```diff
@@ -43,6 +43,16 @@ var (
 			importPubKeyCommand,
 		},
 	}
+
+	// addressesCommand is a wallet subcommand that is responsible for
+	// address management operations.
+	addressesCommand = cli.Command{
```

Comment:
This commit is pretty big. I think we should split it into three parts:
1. All changes to the wallet and wallet controller mock
2. All changes to the RPC
3. All changes in `lncli`

Reply:
Sure!